mojave/jobs/job_types/town/prospector.dm

```diff
@@ -0,0 +1,28 @@
+/datum/job/ms13/town/prospector
```

Comment:
Fairly certain we nixed Prospector after discussion. I'd also advise adjusting job slots to fit more in line with [this updated role distribution doc. ](https://docs.google.com/document/d/1Y1RNRVjfVDMbGqClHRFXWnGCZsf39A2rRSlP6KYBPwk/edit?usp=sharing)